webknossos/webknossos/geometry/bounding_box.py

```diff
@@ -32,6 +33,8 @@ class BoundingBox:
 
     topleft: Vec3Int = attr.field(converter=Vec3Int)
     size: Vec3Int = attr.field(converter=Vec3Int)
+    axes: Tuple[str, str, str] = attr.field(default=("x", "y", "z"))
```

Comment:
Yes would be possible. I also discussed this shortly with @normanrz. The main issue I currently see would be initialization of the old BoundingBox classes. It is currently often done as BoundingBox((0,0,0),(10,10,10)) ... At this point I need to assume that the channel is always at position 0 (as we do currently) and how many channels there are. The bounding box doesn't know about the num_channels property of the layer.

Reply:
We discussed the possible decisions about the magic channel axis yesterday. For now, it will remain as it is for backwards compatibility. But a concept to integrate the channel axis in the bounding box would be great.